WordPressCom-Analytics-iOS/WPAnalytics.h

```diff
@@ -156,7 +156,12 @@ typedef NS_ENUM(NSUInteger, WPAnalyticsStat) {
     WPAnalyticsStatThemesChangedTheme,
     WPAnalyticsStatTwoFactorCodeRequested,
     WPAnalyticsStatTwoFactorSentSMS,
-    WPAnalyticsStatMaxValue
+    WPAnalyticsStatMaxValue,
+    WPAnalyticsStat3DTouchShortcutLogInPressed,
```

Comment:
@kwonye - We can probably drop the "Pressed" here given there's not many other options with 3D touch.